actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -90,18 +100,18 @@ class ActionCable.Connection
     open: ->
       ActionCable.log("WebSocket onopen event")
       @disconnected = false
```

Comment:
Also I found a bug that hadn't been caught before which was that subscription requests were fired off immediately on connecting, even if the server had not sent welcome yet. In some scenarios that would produce odd behavior I imagine.